README.md

```diff
@@ -0,0 +1,136 @@
+### Notebook Images
+
+These images were created to be used with Open Data Hub (ODH) or Red Hat Openshift Data Science (RHODS) with the Kubeflow Notebook Controller as the launcher.
+
+The notebooks support CPU and GPU running modes. GPU notebooks can be also spawned in clusters without GPU enabled using the CPU capabilities:
+
+**CPU Notebooks**
+- Minimal
+- Standard Data Science
+
+**GPU Notebooks**
+- CUDA
+- CUDA PyTorch
+- CUDA TensorFlow
+
+
+#### Folder Structure of the Repository
+
+```
+.
+├── base
+│   └── ubi8-python-3.8
+│       └── Dockerfile		#Includes the instructions for the base image
+├── LICENSE
+├── Makefile                #Lists shell commands for the build and push process
+├── OWNERS                  #Includes the mainteiners of the repository
+└── README.md               #Includes basic information about the repository
+
+```
+
```

Comment:
I don't think this section is needed, the repository structure is something that changes very often and it is hard to maintain.